Backend_Mapanimal/obj/Debug/net5.0/Backend_Mapanimal.RazorTargetAssemblyInfo.cache

```diff
@@ -0,0 +1 @@
+20c96c348501274af84739c00da09e0ec30b2bd7
```

Comment:
Please add a .gitignore file to skip unnecessary files to be uploaded, and please remember to remove those ones in the next commit.